examples/basic/client/src/main.rs

```diff
@@ -6,56 +6,73 @@ use com::{
 use interface::{Food, IAnimal, ICat, IDomesticAnimal, IExample, CLSID_CAT_CLASS};
 
 fn main() {
+    // Initialize the COM apartment
     init_apartment(ApartmentType::SingleThreaded)
         .unwrap_or_else(|hr| panic!("Failed to initialize COM Library{:x}", hr));
+    println!("Initialized apartment");
 
+    // Get a `BritishShortHairCat` class factory
     let factory = get_class_object::<IClassFactory>(&CLSID_CAT_CLASS)
         .unwrap_or_else(|hr| panic!("Failed to get cat class object 0x{:x}", hr));
     println!("Got cat class object");
 
+    // Get an instance of a `BritishShortHairCat` as the `IUnknown` interface
     let unknown = factory
         .get_instance::<IUnknown>()
```

Comment:
Why is `CreateInstance` renamed to `get_instance`?

Reply:
This is because there is no method overloading. I decided to rename the safe functions since they have different signatures and thus calling the safe version `create_instance` might be confusing. I'm starting to think that for the raw win32 COM API calls we should use the CamelCase names and the safe versions can use the snake_case convention. This might be a better way to distinguish the two. Thoughts?

Reply:
Yeah, I don't renaming is such a good idea. It implies they do something else.